server/methods/core/orders.js

```diff
@@ -139,6 +139,18 @@ Meteor.methods({
       throw new Meteor.Error(403, "Access Denied");
     }
 
+    // Server-side check to make sure discount is not greater than orderTotal.
+    let orderTotal =
+      order.billing[0].invoice.subtotal
+      + order.billing[0].invoice.shipping
+      + order.billing[0].invoice.taxes;
+
+    if (discount > orderTotal) {
+      const err = "Discount is greater than the cart total";
+      Logger.error(err);
+      throw new Meteor.Error("orders/approvePayment.discount-amount", err);
```

Comment:
I thought we had settled this discussion previously (error object, error string) for Logger?  I thought this was what @jshimko had proposed.  (otherwise we just used a string). The http status code were traditionally for `throw new Meteor.error` not `Logger`.